src/nvim/eval.c

```diff
@@ -6531,13 +6531,18 @@ static void f_abs(typval_T *argvars, typval_T *rettv, FunPtr fptr)
 static void f_add(typval_T *argvars, typval_T *rettv, FunPtr fptr)
 {
   list_T      *l;
+  const char *const arg_errmsg = N_("add() argument");
 
   rettv->vval.v_number = 1;   /* Default: Failed */
   if (argvars[0].v_type == VAR_LIST) {
     if ((l = argvars[0].vval.v_list) != NULL
         && !tv_check_lock(l->lv_lock, "add() argument", TV_TRANSLATE)) {
```

Comment:
(And that other place is missing N_() for some reason. Most likely an artifact from a massive non-automated refactoring performed by me.)

Reply:
Depending on the answers to the questions above, I'll add the N_, or change this line.